frontend/src/pages/geography/[geographyId].tsx

```diff
@@ -0,0 +1,233 @@
+        ? renderEmpty("Executive")
+        : summary.top_documents.Policy.map((doc) => (
+            <div key={doc.document_id} className="mt-4 mb-10">
+              <RelatedDocumentFull document={doc} />
+            </div>
+          ));
+    }
+    // Legislative
+    if (selectedCategoryIndex === 2) {
+      return summary.top_documents.Law.length === 0
+        ? renderEmpty("Legislative")
+        : summary.top_documents.Law.map((doc) => (
+            <div key={doc.document_id} className="mt-4 mb-10">
+              <RelatedDocumentFull document={doc} />
+            </div>
+          ));
+    }
+    // Litigation
+    if (selectedCategoryIndex === 3) {
+      return (
+        <div className="mt-4">
+          Climate litigation case documents are coming soon to Climate Policy Radar. In the meantime, head to{" "}
+          <a className="text-blue-500 transition duration-300 hover:text-indigo-600" href="https://climate-laws.org/litigation_cases" target={"_blank"}>
+            climate-laws.org/litigation_cases
+          </a>
+        </div>
+      );
+    }
+  };
+
+  useEffect(() => {
+    if (router.query.geographyId) {
+      refetchGeography();
+      refetchSummary();
+    }
+  }, [router.query.geographyId, refetchGeography, refetchSummary]);
+
+  let targets = [];
+  if (!!summary?.targets) targets = showAllTargets ? summary.targets : summary.targets.slice(0, TARGETS_SHOW);
+
+  if (isFetching || isFetchingSummary) return <Loading />;
+
+  return (
+    <>
+      <Layout title={`Climate Policy Radar | ${geography?.name ?? "Loading..."}`}>
+        {isError || !geography ? (
+          <SingleCol>
+            <TextLink onClick={() => router.back()}>Go back</TextLink>
+            <p>We were not able to load the data for the country.</p>
+          </SingleCol>
+        ) : (
+          <section className="mb-8">
+            <CountryHeader country={geography} />
+            <SingleCol>
+              <section className="grid grid-cols-2 md:grid-cols-3 gap-px rounded mb-8">
+                {<KeyDetail detail="Executive" extraDetail="Policies" amount={summary.document_counts.Policy} icon={<PolicyIcon />} />}
+                {<KeyDetail detail="Legislative" extraDetail="Laws, Decrees" amount={summary.document_counts.Law} icon={<LawIcon />} />}
+                {<KeyDetail detail="Litigation" extraDetail="Cases" amount={summary.document_counts.Case} icon={<CaseIcon />} />}
+              </section>
+              {hasEvents && (
+                <section className="mt-12">
+                  <h3 className="mb-4">Events</h3>
+                  <Timeline>
+                    {summary.events.map((event: TEvent, index: number) => (
+                      <Event event={event} key={`event-${index}`} index={index} last={index === summary.events.length - 1 ? true : false} />
+                    ))}
+                  </Timeline>
+                </section>
+              )}
+
+              {hasTargets && (
```

Comment:
Should/could this be a separate "Target" component?

Reply:
I've kept the `<Targets />` component code and typing in the same file for now because it is all driven by dummy data and will almost certainly change. Useful for when the target data comes along and whoever does the development can quickly refer to the component code.

Reply:
sgtm